tools/src/test/java/org/apache/kafka/tools/reassign/ReassignPartitionsUnitTest.java

```diff
@@ -300,6 +301,15 @@ public void testGetReplicaAssignments() throws Exception {
 
             assertEquals(assignments,
                 getReplicaAssignmentForPartitions(adminClient, new HashSet<>(asList(new TopicPartition("foo", 0), new TopicPartition("bar", 0)))));
+
+            assignments.clear();
```

Comment:
Sorry, what's the purpose we explicitly clear `assignments` here? I think this is only used as local scope variable, we should not clear it.

Reply:
You're right, already removed it in the latest commit, many thanks :smiley: